### ISSUE TITLE
Project Status

### ISSUE THREAD

**Issue by sudeepdino008** [rs-ipfs/rust-ipfs#461]
Hi, I'm interested in contributing to the rust-ipfs project. Is the project active? - noticed that the last commit was 5 months ago.

**Comment by aphelionz** [comment]
Yes, it has been a long time since the last commit but that's not for lack of wanting :) We are simply "between funding rounds" at the moment so the project is taking a bit of a nap.

If you're looking to contribute, please check out the "help wanted" and "good first issue" tags as a place to start and let us know if you have any questions as you go through them!

@koivunej anything else to add?

**Comment by gilescope** [comment]
Did you get some more funding?
